lib/python/rose/apps/rose_ana.py

```diff
@@ -196,11 +197,15 @@ def run(self, app_runner, conf_tree, opts, args, uuid, work_files):
             return old_app.run(
                 app_runner, conf_tree, opts, args, uuid, work_files)
 
+        # Load any rose_ana specific configuration settings either from the
+        # site defaults or the user's personal config
+        self._get_global_ana_config()
+
         # If the user's config indicates that it should be used - attach
         # to the KGO database instance in case it is needed later.
-        use_kgo = self.rose_conf.get_value(["rose-ana", "kgo-database"])
+        use_kgo = self.ana_config.get("kgo-database", ".false.")
         self.kgo_db = None
-        if use_kgo is not None and use_kgo == ".true.":
+        if use_kgo == ".true.":
```

Comment:
Good spot, I'll borrow from there (don't need `.false.` though as I don't think I ever reference it)